driver/driver.go

```diff
@@ -147,7 +314,28 @@ func (d *ContrailDriver) Join(req *network.JoinRequest) (*network.JoinResponse,
 	for k, v := range req.Options {
 		fmt.Printf("%v: %v\n", k, v)
 	}
-	r := &network.JoinResponse{}
+
+	tenant, netName, err := d.tenantAndNetnameFromDockerNetwork(req.NetworkID)
+	if err != nil {
+		return nil, err
+	}
+
+	contrailNetwork, err := d.controller.GetNetwork(tenant, netName)
+	log.Infoln("Retreived Contrail network:", contrailNetwork.GetUuid())
+	if err != nil {
+		return nil, err
+	}
+
+	gw, err := d.controller.GetDefaultGatewayIp(contrailNetwork)
+	if err != nil {
+		return nil, err
+	}
+
+	r := &network.JoinResponse{
+		DisableGatewayService: true,
```

Comment:
This is done similarly to https://github.com/docker/libnetwork/blob/master/drivers/windows/windows.go#L640